test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -1763,7 +1763,7 @@ public void PushCommand_APIV2Package_Endpoint()
                         "push",
                         packageFileName,
                         "-Source",
-                        server.Uri + "/api/v2/Package"
+                        server.Uri + "api/v2/Package"
```

Comment:
What part of Mono is failing on the double slash? This test seems fine, but we should understand the root cause here. The different in `//` handling seems a bit scary.